code/game/jobs/job/captain.dm

```diff
@@ -186,7 +186,7 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 	minimal_player_age = 5
 	minimal_player_age = 9
 	ideal_character_age = 30
-	alt_titles = list("UIU Relations Agent" = /decl/hierarchy/outfit/job/site90/crew/civ/uiu,"Horizon Initiative Scribe" = /decl/hierarchy/outfit/job/thirep)
+	alt_titles = list("UIU Relations Agent" = /decl/hierarchy/outfit/job/site90/crew/civ/uiu,"Horizon Initiative Scribe" = /decl/hierarchy/outfit/job/, "Marshall, Carter, and Dark Corporate Liaison" = /decl/hierarchy/outfit/job/site90/crew/civ/MCDRep)
```

Comment:
who's the fucking joker that double defined the jobs in both the map job dm and the base game job dm files

Reply:
Bit of a tangent: Cant remember where this started originally but in total most jobs on Baycode are defined in 2-3 locations that do not point to eachother with comments or even reference fucking eachother directly. Manny probably knows (HoH dev if you don't know them). Not that I'd recommend asking over something trivial like that. Not having it defined in two places used to cause issues (probably does here too) and I'm saying all this without having glanced at SCP13s code more than once or twice.

Reply:
While it may seem redundant, the entire thing breaks if you don't double define it.

Reply:
> While it may seem redundant, the entire thing breaks if you don't double define it.

yeah it's not your fault this is just a general question, wondering which joker did this